src/linter.rs

```diff
@@ -36,7 +36,7 @@ pub struct Context {
 }
 
 impl Context {
-  pub(crate) fn add_diagnostic(
```

Comment:
This was earlier private to the crate itself - had to make it public for external plugins